### PR TITLE
Fix prerelease deployment of maps-db migration files

### PR DESCRIPTION
**Pull request by DanVanAtta**
1. We were not copying the maps-db migration zip file for prerelease, this
update fixes that so that the zip file will be available to ansible.

2. Minor, update the name of the maps migration file for clarity:
'map_migration.zip' -> 'maps_database_migrations.zip'

## Functional Changes

[] New map or map update
[] New Feature
[] Feature update or enhancement
[] Feature Removal
[] Code Cleanup or refactor
[] Configuration Change
[x] Problem fix
[] Other:

### CODE REVIEW

**Comment by DanVanAtta** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Coverage decreased per file
===========================
- game-core/src/main/java/games/strategy/net/nio/Decoder.java  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/triplea-game/triplea/pullRequest?prid=5947078&bid=19495525)

File: infrastructure/ansible/roles/database/flyway/defaults/main.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ flyway_args:
     password: "{{ maps_db_password }}"
     url: "{{ flyway_db_url }}/{{ maps_db_name }}"
     migration:
-      zip: map_migrations.zip
+      zip: maps_database__migrations.zip
```
Comment (RoiEXLab) [inline infrastructure/ansible/roles/database/flyway/defaults/main.yml:19]: Double underscore? I'm pretty sure this isn't intentional.

Comment (DanVanAtta) [reply infrastructure/ansible/roles/database/flyway/defaults/main.yml]: Indeed also on line 20 the name was not updated either: map_migrations.zip

Both fixed in: d594b933a

**Comment by DanVanAtta** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Coverage decreased per file
===========================
- game-core/src/main/java/games/strategy/triplea/ui/unit/scroller/AvatarCoordinateCalculator.java  -8
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/triplea-game/triplea/pullRequest?prid=5947078&bid=19495525)

**Pull request merged**